apiv2/platformics/graphql_api/core/strawberry_helpers.py

```diff
@@ -0,0 +1,41 @@
+from typing import Tuple
+
+from strawberry.types.nodes import SelectedField
+
+from platformics.graphql_api.core.errors import PlatformicsError
```

Comment:
This file basically adds some helper functions that filter out graphql reserved fields (they start with `__`) from a list of fields in a query.